subworkflows/local/prepare_alphafold2_dbs.nf

```diff
@@ -18,96 +18,127 @@ include { COMBINE_UNIPROT   } from '../../modules/local/combine_uniprot'
 include { DOWNLOAD_PDBMMCIF } from '../../modules/local/download_pdbmmcif'
 
 workflow PREPARE_ALPHAFOLD2_DBS {
+
+    take:
+    alphafold2_db            // directory: path to alphafold2 DBs
+    full_dbs                 //  boolean: Use full databases (otherwise reduced version)
```

Comment:
Being picky here, the comments are not well aligned 😄

Reply:
Done in [0fffa5f](https://github.com/nf-core/proteinfold/pull/160/commits/0fffa5f518c497de65b5749eaadc3b9cc1bd92b7)